flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/GroupWindowITCase.scala

```diff
@@ -18,6 +18,8 @@
 
 package org.apache.flink.table.planner.runtime.stream.sql
 
+import java.math.BigDecimal
```

Comment:
nit: reorder the imports

Reply:
@godfreyhe Thanks for the review, I have addressed all your comments, including add UT and IT about window aggregate consumes upsert input stream. please have a look, Thanks.
BTW, an unexpected exception is thrown out when add IT for window aggregate consumes upsert input stream. I create an issue [FLINK-22680](https://issues.apache.org/jira/browse/FLINK-22680) for the bug. And I only add a simple ITCase in `GroupWindowITCase.scala` which could avoid the bug in this pr.